code/modules/food_and_drinks/kitchen_machinery/deep_fryer.dm

```diff
@@ -49,6 +49,9 @@ insert ascii eagle on american flag background here
 	if(istype(I, /obj/item/reagent_containers/food/snacks/deepfryholder))
 		to_chat(user, "<span class='userdanger'>Your cooking skills are not up to the legendary Doublefry technique.</span>")
 		return
+	if(istype(I,/obj/item/clothing/head/mob_holder))
+		to_chat(user, "<span class='warning'>What the fuck is wrong with you? No!</span>")
```

Comment:
I think it's a bit rude. Plus, you should totes be able to fry people with an emagged fryer. Perhaps let them leave if they can, but otherwise absolutely wreck them? It's not much worse than a gibber or just choking them to death if they already can't move.

Reply:
I'll make this less rude, but I think Washing Machine/Deep Fryer interactions for mob_holder is going to have to come later. It's something of a balance question, too, that I'm unprepared to answer at the moment